src/libraries/System.Drawing.Common/ref/System.Drawing.Common.csproj

```diff
@@ -34,4 +39,7 @@
     <Reference Include="System.Runtime.Extensions" />
     <Reference Include="System.Runtime.InteropServices" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+    <PackageReference Include="System.Numerics.Vectors" Version="$(SystemNumericsVectorsVersion)" />
```

Comment:
Why is this needed?  I noticed this dependency appears as new in the diff. Was API added that needs that assembly?

Reply:
Yes. https://github.com/dotnet/runtime/commit/667b4e7729c78208ae1e38a03ea38e4ba3c83acc#diff-7fa436bd8d0d91db628f7f87a6c9bfd69bc13d00ec0567eeb738312dcaecbb75

Reply:
That API can’t be present on .NETFramework, shouldn’t we ifdef it out of the `netstandard2.0` assembly?

Reply:
Yes, I think we should hide it from .NET Framework and .NET Standard 2.0 ref assembly.

Reply:
Factored that API out into a separate file. Updated the nuspec diff.